components/UploadedFilePreview.js

```diff
@@ -87,6 +87,19 @@ const PrivateIconContainer = styled.div`
   }
 `;
 
+const Message = () => injectIntl(({ id, intl }) => intl.formatMessage(id));
```

Comment:
Why create a custom component? It seems to me that `FormattedMessage` does the same thing